### PR TITLE
Add MSU Federal Credit Union

### PR DESCRIPTION
**Pull request by mcvic1rj**
Add entry for msufcu.org 
Rank#: 58,885 
Icon is a PNG as there is not a publicly available SVG

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The build fails as the PNG isn't 32x32px, 64x64px or 128x128px. I recommend you use this SVG instead:
```SVG
<svg xmlns="http://www.w3.org/2000/svg" style="background:#014e3a;fill:#25b570" viewBox="0 0 85 85"><path d="M50.9 6.3h-3.7L34.1 78.6h3.7zm4.3 5.8L45 67.3h3.6L58.7 12zm-18.8 4.7L26.3 72h3.6l10.2-55.3z"/></svg>
```

**Comment by mcvic1rj** [comment]
Thanks for the SVG @Carlgo11. I have swapped over to it and will see how the tests run.

File: entries/m/msufcu.org.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "domain": "msufcu.org",
+        "tfa": [
+            "sms",
+            "call",
+            "email"
+        ],
+        "documentation": "https://www.msufcu.org/securitycenter",
+        "keywords": [
+            "banking"
+        ]
```
Comment (Carlgo11) [inline entries/m/msufcu.org.json:12]: As only US residents can be customers a regions element is also needed.
```suggestion
        ],
        "regions": [
          "us"
        ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
:warning: Unless there's activity within the next 7 days I'll close the PR.

**Comment by Carlgo11** [comment]
Closing due to inactivity.
> **Note**
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**